payments/invoice/other-features.md

```diff
@@ -196,34 +287,239 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-  "payment": "/psp/invoice/payments/<payments-id>",
+  "payment": {
+    "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
+    "number": 1234567890,
+    "created": "2016-09-14T13:21:29.3182115Z",
+    "updated": "2016-09-14T13:21:57.6627579Z",
+    "operation": "Verify",
+    "state": "Ready",
+    "currency": "NOK",
+    "amount": 0,
+    "description": "Test Verification",
+    "payerReference": "AB1234",
+    "initiatingSystemUserAgent": "PostmanRuntime/3.0.1",
+    "userAgent": "Mozilla/5.0",
+    "language": "nb-NO",
+    "transactions": {
+      "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions"
+    },
+    "verifications": {
+      "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/verifications"
+    },
+    "urls": {
+      "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/urls"
+    },
+    "payeeInfo": {
+      "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/payeeInfo"
+    },
+    "settings": {
+      "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/settings"
+    }
+  },
+  "operations": [
+    {
+      "method": "POST",
+      "href": "https://api.externalintegration.payex.com/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/approvedlegaladdress",
+      "rel": "create-approved-legal-address",
+      "contentType": "application/json"
+    },
+    {
+      "method": "POST",
+      "href": "https://api.externalintegration.payex.com/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/authorizations",
+      "rel": "create-authorization",
+      "contentType": "application/json"
+    },
+    {
+      "method": "PATCH",
+      "href": "https://api.externalintegration.payex.com/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
+      "rel": "update-payment-abort",
+      "contentType": "application/json"
+    },
+    {
+      "method": "GET",
+      "href": "https://ecom.externalintegration.payex.com/invoice/payments/authorize/2f9b51a821d40dd015332f14460f91393856725a19e9fb5a834d460af91c9ce2",
+      "rel": "redirect-authorization",
+      "contentType": "text/html"
+    }
+  ]
+}
+```
+
+### Verification flow
+
+The sequence diagram below shows the two requests you have to send to Swedbank
+Pay to make a purchase. The links will take you directly to the API description
+for the specific request. The diagram also shows in high level, the sequence of
+the process of a complete purchase.
+When dealing with credit invoice payments, 3-D Secure authentication of the
+cardholder is an essential topic. There are three alternative outcome of a
+credit invoice payment:
+
+* 3-D Secure enabled - by default, 3-D Secure should be enabled, and Swedbank
```

Comment:
Please delete all mentions of "3-D Secure".

Reply:
Fixed.